lib/mobility-core/src/Kernel/Types/Beckn/City.hs

```diff
@@ -95,6 +98,7 @@ instance ToJSON City where
   toJSON Goa = String "std:08342"
   toJSON Pune = String "std:020"
   toJSON Mysore = String "std:0821"
+  toJSON Tumkuru = String "std:0816"
```

Comment:
![image](https://github.com/nammayatri/shared-kernel/assets/15854798/8502cdd9-d430-497c-b49b-a6ddb2a2e7ac)
Its `Tumakuru`, not `Tumkuru`

Reply:
Ohh resolving